tools/metrics/histograms/metadata/cobalt/histograms.xml

```diff
@@ -119,15 +119,14 @@ Always run the pretty print utility on this file after editing:
   </summary>
 </histogram>
 
-<histogram name="Cobalt.Media.HasEverPlayed" enum="PipelineStatus"
+<histogram name="Cobalt.Media.HasEverPlayed" enum="BooleanHasPlayed"
```

Comment:
I _think_ changing the enum definition on an existing histogram is mostly safe... but one issue will be that historical data of Cobalt.Media.HasEverPlayed will always use the old PipelineStatus labels. There will be a period time where both sets of labels show up in the dashboard (you'll see 4 unique values), but it should fix itself after the pipeline runs for a week or so. But if you ever query for older dates, you'll see these old labels.

Reply:
ack!